src/main/java/ua/softserveinc/tc/controller/admin/DayOffController.java

```diff
@@ -53,17 +57,18 @@
 
         DayOff currentDay = dayOffService.findById(id);
         if (currentDay == null) {
-            System.out.println("While getting user with id " + id + " not found");
+            log.error("While getting user with id " + id + " not found");
             return new ResponseEntity<>(HttpStatus.NOT_FOUND);
         }
         return new ResponseEntity<>(currentDay, HttpStatus.OK);
     }
 
     @PostMapping("/day/")
     public ResponseEntity<Void> createDayOff(@RequestBody DayOff dayOff, UriComponentsBuilder ucBuilder) {
-        if (dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
+        if (!dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
```

Comment:
seems like "dayOffExist" returns true if day off acrually doesnt exists - i'd rename that method